src/routes/routes.tsx

```diff
@@ -0,0 +1,15 @@
+import { Route } from "react-router-dom";
+import { BooksPage } from "../domain/books/BooksPage";
+import { CartPage } from "../domain/cart/CartPage";
+
+const createRoutes = () => {
```

Comment:
Just rename it to Routes or Router whatever you like and import it in App like a react component